src/core/Akka.Remote/Endpoint.cs

```diff
@@ -1085,18 +1086,7 @@ protected override void PreStart()
         {
             if (_handle == null)
             {
-                Transport
-                    .Associate(RemoteAddress, _refuseUid)
-                    .ContinueWith(handle =>
-                    {
-                        if (handle.IsFaulted)
-                        {
-                            var inner = handle.Exception?.Flatten().InnerException;
-                            return (object)new Status.Failure(new InvalidAssociationException("Association failure", inner));
-                        }
-                        return new Handle(handle.Result);
-                    }, CancellationToken.None, TaskContinuationOptions.ExecuteSynchronously, TaskScheduler.Default)
-                    .PipeTo(Self);
+                AssociateAsync().PipeTo(Self);
```

Comment:
Same story here - simplification in order to get rid of AggregatedExceptions. I am fully following PipeTo guideline here.